test/lib/ufe/testMaterial.py

```diff
@@ -78,9 +80,14 @@ def testUsdNoMaterial(self):
         cubeItem = ufe.Hierarchy.createItem(cubePath)
 
         materialInterface = ufe.Material.material(cubeItem)
+
         materials = materialInterface.getMaterials()
         self.assertEqual(len(materials), 0)
 
+        if os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') >= '5002':
```

Comment:
Having tests with specific version related sections is frowned upon. Please duplicate the whole test, and use `@unittest.skipIf` decorators to enable one or the other depending on the UFE version.

Reply:
cool, changed! thanks Jerry